### PR TITLE
Dev week3

### PR DESCRIPTION
**Pull request by FormerlyZeroCool**

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -123,24 +123,25 @@ function jsListToHtml(list){
 
     for(element of list)
     {
-      const checked = element.status=="closed"?'checked':"";
+      const checked = element.status==="closed"?'checked':"";
       html += `<input type="checkbox" name="${element.id}" value="${element.message}" ${checked} 
                 onclick="updateRecordStatus(event.target)"/>
-      <label for="${element.id}">${element.message}</label><br><br>`;
+      <label for="${element.id}">${element.message}</label>
```
Comment (lyteffera) [inline src/main/webapp/script.js:52]: I'm a little confused by the syntax here, could you explain what's going on here in a little more detail?

**Review by lyteffera — APPROVED** [review]
Looks good!

File: src/main/java/com/calendarfiles/TaskProcessor.java [hunk 2]
```diff
@@ -44,7 +44,7 @@
     private ArrayList<Event> eventList; 
 
     public TaskProcessor(){
-        datastore = DataStoreServiceFactory.getDataStoreService(); 
+        //datastore = DataStoreServiceFactory.getDataStoreService();
```
Comment (lyteffera) [inline src/main/java/com/calendarfiles/TaskProcessor.java:47]: I'm not sure if this is currently a comment, but if so would recommend getting rid of this line of code.

**Review by AzfaarQureshi — APPROVED** [review]
Looks good to me!

**Pull request merged**